frontend/src/pages/CompareV2.tsx

```diff
@@ -475,14 +541,17 @@ function CompareV2(props: CompareV2Props) {
             onSwitch={setMetricsTab}
           />
           <div className={classes(padding(20, 'lrt'), css.outputsOverflow)}>
-            {metricsTab === MetricsType.SCALAR_METRICS &&
-              (scalarMetricsTableData ? (
-                <CompareTable {...scalarMetricsTableData} />
-              ) : (
-                <p>There are no Scalar Metrics artifacts available on the selected runs.</p>
-              ))}
+            {metricsTab === MetricsType.SCALAR_METRICS && (
+              <ScalarMetricsTable
+                isErrorArtifacts={isErrorArtifacts}
+                isLoadingArtifacts={isLoadingArtifacts}
+                scalarMetricsTableData={scalarMetricsTableData}
```

Comment:
NIT:
When the `isErrorArtifacts` is true, I think we can replace whole `Metrics section (line 529)` with the error statement, because the result across all sub-sections will be the same (empty page) when the backend call is loading or failing.

Reply:
This change makes sense - I think this applies for the `isLoadingArtifacts` as well, so I've updated it accordingly. This simplified the logic quite a bit, so thank you! https://github.com/kubeflow/pipelines/pull/8029/commits/9d6a22202511b482d2fa2c26ceb194b74a36c4e0

I considered moving this whole `Metrics section (line 529)` to its own function, but since I would have to pass in too many props (all of the different artifact types), I decided against this.